docs/esm-views/customize-bundle-strategy.md

```diff
@@ -7,7 +7,8 @@ title: Customize bundling strategy
 # Customize bundling / rewriting strategy
 
 By default, all external dependencies are rewritten to a CDN URL and none is
-bundled. This logic can be controlled using two environment variables:
+bundled. This logic can be controlled using two environment variables or by
+using a [modular configuration file](../configuration.md).:
 
 1. `EXTERNAL_ALLOW_LIST` is a comma-separated string that specifies which
```

Comment:
You could link to the specific option here...

```suggestion
1. [EXTERNAL_ALLOW_LIST](../configuration.md#externalallowlist) is a comma-separated string that specifies which
```

Reply:
Good suggestion